server/report_builder/README.md

```diff
@@ -1,464 +0,0 @@
-
-```yaml
-url: "https://demo-open.msupply.org"
-# standard omSupply username and password
-username: "username"
-password: "password"
-```
-
-The report builder will use this config file to authenticate with the remote-server and print/download the report to the local file system.
-The remote-server needs a store id (or store name) and a data id to print the report.
-For example, to print a report for a stocktake with id "d734fd45-064e-4ddd-9886-ea71a2797640" from store "80004C94067A4CE5A34FC343EB1B4306":
-
-```bash
-> report_builder print --report generated/output.json --config config.yaml --store-id 80004C94067A4CE5A34FC343EB1B4306 --data-id d734fd45-064e-4ddd-9886-ea71a2797640 --output report_pdf_name.pdf
-```
-
-If the store name instead of the store id is provided, the matching store id is automatically fetched from the central server.
-For example, the following works as well:
-
-```bash
-> report_builder print --report generated/output.json --config config.yaml --store-name "Gryffindor District Store" --data-id d734fd45-064e-4ddd-9886-ea71a2797640 --output report_pdf_name.pdf
-```
-
-### Report templates with arguments
-
-Some reports need additional arguments such as a time range.
-Report arguments are passed on as variable to the used GraphGl query.
-
-There is an example for a report with arguments in the `example_arguments` directory.
-To specify the arguments create a `arguments.json` file in the `report_builder` directory with the content:
-
-```json
-{
-  "creationDatetimeBeforeOrEqual": "2023-01-16T01:35:39.770Z"
-}
-```
-
-The required arguments are the same as the variables one would pass to the underlying GraphGl query,
-i.e. to learn about available parameters and types please look at the GraphGl query definition.
-
-To build the report run:
-
-```bash
-report_build -- build --dir ./example_arguments/ --template template.html --query-gql query.graphql
-report_build -- print --report generated/output.json --config config.yaml --store-id 80004C94067A4CE5A34FC343EB1B4306 --arguments-file ./arguments.json --output report_pdf_name.pdf
-```
-
-## References to other template definitions
-
-It's possible to refer to other template resources that already exist on the server, e.g. to refer to a common headers or icons.
-
-To refer to a resource from an existing report add a reference file like `icon1.ref.json` to the project with the content:
-
-```json
-{
-  "source": "existing_report_id"
-}
-```
-
-The existing report must contain an entry of name `icon1`.
-The template entry `icon1` can then be used in the Tera templates the same as any other entry.
-
-If there is a name conflict it is possible to rename referred template entries.
-For example, instead of `icon1` an alternative name can be used by using a reference file like `my_icon_name.ref.json`:
-
-```json
-{
-  "source": "existing_report_id",
-  "sourceName": "icon1"
-}
-```
-
-The entry `icon1` from the existing report can then be used under the name `my_icon_name`.
```

Comment:
This is the only section I wasn't sure about in the old report readme.

Are we using this functionality anywhere? I couldn't see in any of our reports.

Maybe we need to update our test-config.json and/or report-manifest.json files to ensure we still have access to this functionality? 

I have not yet raised an issue for this.

Reply:
Not to sure sorry =/